server/covmanager/templates/reportconfigurations/summary.html

```diff
@@ -61,66 +61,56 @@
 
 let pmanager = new HashParamManager()
 
-let covmanager = new Vue({
-  el: '#main',
+const app = Vue.createApp({
   components: {
     vueSlider,
   },
-  data: {
-    csrftoken: "",
-    loading: false,
-    loading_incomplete: false,
-    chart: {
-      data: {},
-      level: 0,
-      maxlevel: 0,
-      slider: {
-        initialized: false,
-      },
-    }
-  },
-  watch: {
-    'chart.data': function() {
-      this.draw()
-    },
-    'chart.level': function() {
-      this.draw()
-    },
-    'chart.maxlevel': function() {
-      if (!this.chart.slider.initialized) {
-        this.chart.level = this.chart.maxlevel
-        this.chart.slider.initialized = true
-      }
-    }
+  data() {
+    return {
+      csrftoken: "",
+    };
   },
-  created: function () {
+  created(){
     this.csrftoken = getCSRFToken()
     this.fetch()
 
     const formattedDate = formatClientTimestamp("{{collection.created}}")
     this.chart.data.created = formattedDate
     document.title += ` - ${formattedDate}`
   },
-  methods: {
-    fetch: _.throttle (function () {
-      this.loading = true
-      fetch(URLS.rs_api, {
+  setup() {
+    const loading = Vue.ref(false)
+    const loading_incomplete = Vue.ref(false)
+    const chart = Vue.reactive({
+      data: {},
```

Comment:
This is not working in the PR. Please see updated test data for example.

![image](https://github.com/user-attachments/assets/ad7f28df-8837-49e4-94a7-6404e91619f2)